src/pages/AccessReview.js

```diff
@@ -37,8 +41,9 @@ class AccessReview extends Component {
             return prev;
           });
         }).catch(error => {
-          this.setState({ showConfirmationDialogOK: true, alertMessage: "Sorry, something went wrong when trying to submit the vote. Please try again." });
-        });
+        this.setState(
```

Comment:
indentation ?

Reply:
Yes, should be indented. Intellij's auto-formatter seems to be moving these back after make the changes. Will look at that setting.

Reply:
Actually, no, the auto-formatter is correct here. It's confusing to the eye due to the previous line, `}).catch(error => {`. I'll update the blocks so it looks better and the formatter works.